packages/gatsby-source-contentful/src/plugin-options.js

```diff
@@ -112,6 +117,23 @@ const formatPluginOptionsForCLI = (pluginOptions, errors = {}) => {
   return lines.join(`\n`)
 }
 
+const initializeErrorMap = ({ reporter }) => {
+  if (reporter?.setErrorMap) {
+    reporter.setErrorMap({
+      TODO_NUMBER: {
+        text: context =>
+          `Problems with gatsby-source-contentful plugin options: ${formatPluginOptionsForCLI(
+            context.options,
+            context.errors
+          )}`,
+        errorType: `USER`,
+        level: `ERROR`,
+        docsUrl: `https://www.gatsbyjs.org/docs/gatsby-cli/#new`,
```

Comment:
I'm not sure why this is the url for this error. Is it a placeholder? (The field is optional so perhaps don't set it here at all?)